src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Button from "../Button/Button";
+import "./_navbar.scss";
+import "../Button/_button.scss";
+
+function Navbar() {
+  return (
+    <nav className="navbar">
+      <div className="navbar-content">
+        <p className="name">JobsPortal</p>
+        <div className="button-container2">
+          <Button title="Login" className="button-container" />
```

Comment:
ne treba proslijedjivati className ovdje jer ti Button komponenta ne prima className kao prop. Dovoljno je sto je jednom taj className naveden u Button komponenti da to radi